content/hi/docs/setup/best-practices/cluster-large.md

```diff
@@ -0,0 +1,88 @@
+---
+title: बड़े क्लस्टर्ज़ के लिए विचार
+weight: 20
+---
+
+एक क्लस्टर {{< glossary_tooltip text="नोड्स" term_id="node" >}} (भौतिक या वर्चुअल मशीन) का एक सेट है जो कुबेरनेट्स एजेंटों को चला रहा है, जिसे {{< glossary_tooltip text="कण्ट्रोल प्लेन" term_id="control-plane" >}} द्वारा प्रबंधित किया जाता है|
+कुबेरनेट्स {{< param "version" >}} 5000 नोड्स तक क्लस्टर का समर्थन करता है| अधिक विशेष रूप से, कुबेरनेट्स को ऐसे कॉन्फ़िगरेशन को समायोजित करने के लिए डिज़ाइन किया गया है जो निम्नलिखित *सभी* मानदंडों को पूरा करते हैं: 
+
+* प्रति नोड 110 पॉड से अधिक नहीं
```

Comment:
While I know this is a transliteration off Google, we might need to tune the recommendations to actually make sense. Thoughts, @anubha-v-ardhan ?